src/openrct2/world/TileInspector.cpp

```diff
@@ -577,12 +578,12 @@ int32_t tile_inspector_surface_toggle_diagonal(int32_t x, int32_t y, int32_t fla
 
     if (flags & GAME_COMMAND_FLAG_APPLY)
     {
-        surfaceElement->properties.surface.slope ^= TILE_ELEMENT_SLOPE_DOUBLE_HEIGHT;
-        if (surfaceElement->properties.surface.slope & TILE_ELEMENT_SLOPE_DOUBLE_HEIGHT)
+        surfaceElement->AsSurface()->SetSlope(surfaceElement->AsSurface()->GetSlope() ^ TILE_ELEMENT_SLOPE_DOUBLE_HEIGHT);
```

Comment:
I'd prefer non-trivial arguments to be computed in a temporary variable on their own line.